src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PropertyPages/xlf/BuildPropertyPage.VisualBasic.xaml.zh-Hans.xlf

```diff
@@ -304,43 +304,43 @@
       </trans-unit>
       <trans-unit id="EnumValue|OptionCompare.Binary|DisplayName">
         <source>Binary</source>
-        <target state="translated">二进制</target>
-        <note />
+        <target state="translated">Binary</target>

```

Comment:
this probably is going to be reverted to the previous value; however, I expect translators will see the locked note and know that the source should not be translated.